google-cloud-spanner/src/test/java/com/google/cloud/spanner/InstanceAdminClientImplTest.java

```diff
@@ -379,7 +413,96 @@ public void testUpdateInstanceProcessingUnits() throws Exception {
     when(rpc.updateInstance(
             instance,
             FieldMask.newBuilder()
-                .addAllPaths(Arrays.asList("display_name", "processing_units", "labels"))
+                .addAllPaths(
+                    Arrays.asList(
+                        "display_name", "autoscaling_config", "processing_units", "labels"))
+                .build()))
+        .thenReturn(rawOperationFuture);
+    OperationFuture<Instance, UpdateInstanceMetadata> operation =
+        client.updateInstance(instanceInfo);
+    assertTrue(operation.isDone());
+    assertEquals(INSTANCE_NAME, operation.get().getId().getName());
```

Comment:
Can we add few more assertions on value of nodecount, processing units and autoscaling config. Applicable at other tests as well.

Reply:
The assertions are probably not needed, because we mock the RPC to return an instance only with processing_units: https://screenshot.googleplex.com/7fF5jTJZ8hbQ2nn. So if we add assertions, the assertions will only be to test the mocking of the RPC, instead of the actual code.